src/internal/migrations/migrations_test.go

```diff
@@ -48,3 +52,54 @@ func TestMigration(t *testing.T) {
 	require.NoError(t, db.GetContext(ctx, &max, `SELECT max(id) FROM migrations`))
 	assert.Equal(t, state.Number(), max)
 }
+
+func TestEnterpriseConfigMigration(t *testing.T) {
```

Comment:
This probably isn't a good place for this test.  Now the migrations package depends on enterprise, collections, all stuff it doesn't need. 

You should move this into the enterprise server package.  If you run into dependency issues you can rename the package to `server_test` and Go will compile a separate package, avoiding the circular dependency.

Reply:
Otherwise this test looks good though.